docs/reference/docs/bulk.asciidoc

```diff
@@ -131,7 +131,12 @@ operation based on the `_parent` / `_routing` mapping.
 [[bulk-timestamp]]
 === Timestamp
 
+ifdef::asciidoctor[]
+deprecated[2.0.0-beta2,"The `_timestamp` field is deprecated.  Instead, use a normal <<date,`date`>> field and set its value explicitly"]
```

Comment:
Could you do `deprecated::[` here?

Reply:
Added with 0f487237094e6158495265e4ca64437d99313c52